app/frontend/components/pages/delegations/shelleyBalances.tsx

```diff
@@ -118,11 +112,7 @@ const shelleyBalances = ({
                 : `${printAda(nonStakingBalance)}`}
               <AdaIcon />
             </div>
-            <button
-              disabled={calculatingDelegationFee}
-              className="button secondary convert"
-              onClick={convertNonStakingUtxos}
-            />
+            <button className="button secondary convert" onClick={convertNonStakingUtxos} />
```

Comment:
IMPORTANT: I will revert this part as "delegate" is not yet using "cachedTxPlans" but the old approach, so those checks must stay here. They can only be removed for "DeregisterStakeKey"

Reply:
But it should be fine to review the rest and the concept in general.